doc/source/blog_posts/TRACER.ipynb

```diff
@@ -0,0 +1,2267 @@
+{
```

Comment:
<p>Please add an Imports subheader here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-DOE/pyart/pull/1296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I've added this in the new commit.